### PR TITLE
Added: Neu-GreySun

### PR DESCRIPTION
**Pull request by amauryverschooren**

### CODE REVIEW

File: src/scss/themes/_neugreysun.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+$bgColor: rgb(41, 41, 41);
+$textColor: #fff;
+$lightColor: #333;
+$shadowColor: #000;
+$activeColor: rgb(255, 217, 0);
+$disactiveColor: #777;
+
+@include neutheme.create-neu-theme(
+  "dark",
```
Comment (matteobruni) [inline src/scss/themes/_neugreysun.scss:11]: This name must match the theme in `Editor.ts` file without the `neu-` prefix, please run a `yarn start` to try it

**Review by matteobruni — CHANGES_REQUESTED** [review]

**Review by matteobruni — CHANGES_REQUESTED** [review]
You updated the neudark theme instead of the neugreysun. Please fix it.

Please **test** the changes before submitting.

Just run `yarn && yarn start` in terminal and go to <http://localhost:3030>

**Comment by amauryverschooren** [comment]
Changed the right files this time and new theme works! But when i used "yarn", it says that there are 50 file changes while there are no changes to the files?

**Comment by matteobruni** [comment]
Don't worry about changes, if `yarn` makes them is fine. Prettier and ESLint will fix any code issue they find

**Review by matteobruni — APPROVED** [review]

**Pull request merged**